recipes/msys2/all/conanfile.py

```diff
@@ -11,50 +11,91 @@ class MSYS2Conan(ConanFile):
     homepage = "http://www.msys2.org"
     license = "MSYS license"
     topics = ("conan", "msys", "unix", "subsystem")
-    build_requires = "7zip/19.00"
```

Comment:
I think 7zip was used because python2 does not have the `lzma` module.

Reply:
Yes, please leave this unchanged. We can only change this one day when we only support Conan 2.0+ and therefore Python 3.5+

Reply:
Should I maybe add support for Win 3.11?

Reply:
I think this change should be fine. conan already requires python 3.5+ : https://docs.conan.io/en/latest/installation.html#install-with-pip-recommended